### DISCUSSION TITLE
Unable to calibrate search-strings

### DISCUSSION THREAD

**Discussion by Luphana** [Lailloken/Exile-UI#201] [Q&A]
Version: 1.3.00.1 (first time download)

I wanted to setup search-strings for gwennen and vendors, but I'm unable to calibrate any of those strings.
When I click on calibrate it doesn't seem to save the image into the  img\Recognition folder.
I've already tried to manually add it to the folder as "gwennen" but it didn't fix the problem.

img\Recognition folder:
![Screenshot 2023-03-28 193948](https://user-images.githubusercontent.com/129212386/228323918-b1ecfe2f-017e-4568-9271-670a44605300.jpg)

Calibrate screen-checks does work perfectly fine.

Does someone have a solution for this issue I'm having or is it a bug in 1.3.0.1 (released last week) ?

**Comment by Lailloken** [comment]
I can't look into it at the moment. The screen-capped images for search-strings are tagged with "[search-strings] ", so if you want to manually add the image-file, you'll need to name it "[search-strings] gwennen".

**Reply by Luphana** [reply]
Thanks :) add the image manually with name "[search-strings] gwennen" worked for me 💯
